### DISCUSSION TITLE
Grafana graphs

### DISCUSSION THREAD

**Discussion by seanalvar** [burningalchemist/sql_exporter#495] [General]
Hello, Nice plugin! I was just wondering if anyone has created some grafana graphs for MS sql server? 

Cheers

**Comment by burningalchemist** [comment]
Hi @seanalvar, I guess you can check out this directory - https://github.com/burningalchemist/sql_exporter/tree/master/examples/azure-sql-mi/grafana-dashboard

Cheers!

**Comment by seanalvar** [comment]
It's actually not in azure it's in a private cloud. I'll create some myself just hate graphing :D
Thanks

**Reply by burningalchemist** [reply]
It's all managed by the contributors, as I don't have access to any of Microsoft enterprise products these days. I thought you could use them as an example. 🙂

Feel free to contribute them if you like. It's all highly appreciated. 👍

**Reply by seanalvar** [reply]
What a great idea! I've do some basic ones cheers
